src/hardware/gameblaster.cpp

```diff
@@ -133,21 +133,24 @@ bool GameBlaster::MaybeRenderFrame(AudioFrame &frame)
 	static int16_t *p_buf[]           = {&buf[0], &buf[1]};
 	static device_sound_interface::sound_stream stream;
 
+	int16_t leftAccum = 0;
+	int16_t rightAccum = 0;
+
 	// Accumulate the samples from both SAA-1099 devices
 	devices[0]->sound_stream_update(stream, 0, p_buf, 1);
-	frame.left  = buf[0];
-	frame.right = buf[1];
+	leftAccum = buf[0];
+	rightAccum = buf[1];
```

Comment:
Suggest creating and assign straight way..
`int left_accum = ...`

Reply:
Done!